Resources/doc/reference/configuration/proxy-client.rst

```diff
@@ -236,6 +237,40 @@ HTTP method for sending purge requests to the Symfony HttpCache. Make sure to
 configure the purge plugin for your HttpCache with the matching header if you
 change this.
 
+cloudflare
+-------
+
+.. code-block:: yaml
+
+    # config/packages/fos_http_cache.yaml
+    fos_http_cache:
+        proxy_client:
+            cloudflare:
+                zone_identifier: '<my-zone-identifier>'
+                authentication_token: '<user-authentication-token>'
```

Comment:
```suggestion
                authentication_token: '<user-authentication-token>'
                server: 'https://api.cloudflare.com'
```

Reply:
I've used the same http.servers configuration key/value as the one for Varnish. This is the result of  ``./bin/console d:conf FOSHttpCacheBundle``:

```yaml
    proxy_client:
        cloudflare:
            authentication_token: '%env(CLOUDFLARE_AUTH_TOKEN)%'
            zone_identifier: '%env(CLOUDFLARE_ZONE_ID)%'
            http:
                servers:
                    - 'https://api.cloudflare.com'
                http_client: null
```

Do you think I should remove it from from the ``http``` key and only allow setting one server?

Reply:
ah sorry, no i think using the same structure is good, reduces cognitive overload.

for example, i could configure a client that sends requests to actual cloudflare and some debuggging endpoint i built, to see what is getting invalidated, etc.

Reply:
can you please update the example in the doc to mention the http section?

Reply:
Done in 08e127b6

Reply:
Done in 08e127b6